src/authorizer.js

```diff
@@ -49,6 +49,11 @@ class Authorizer {
 
     let kid = unverifiedToken && unverifiedToken.header && unverifiedToken.header.kid;
 
+    if (!kid) {
+      this.logFunction({ level: 'ERROR', title: 'Unauthorized', details: 'Token did no provide a KID', method: methodArn, token });
```

Comment:
This should likely be `level: 'WARN'`, because it is the expected functionality of the authorizer when the token is invalid.

Reply:
Good point. I've changed that.